lib/panda/error_handler.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'faraday'
+module FaradayMiddleware
+  class PandaErrorHandler < Faraday::Middleware
+    def on_complete(env)
+      raise Panda::APIError.new(env.status, env.body) if env[:status] != 200
```

Comment:
Does `env.status` and `env[:status]` gives different statuses?

Reply:
yes, it's from [here](https://github.com/lostisland/faraday/blob/main/lib/faraday/options/env.rb#L57), but i did't know about that before
I think better to write this condition [`unless env.success?`](https://github.com/lostisland/faraday/blob/main/lib/faraday/options/env.rb#L126-L129)